src/Interpreters/HashJoin.cpp

```diff
@@ -2009,12 +2002,14 @@ struct AdderNonJoined
 /// Based on:
 ///   - map offsetInternal saved in used_flags for single disjuncts
 ///   - flags in BlockWithFlags for multiple disjuncts
-template <bool multiple_disjuncts>
```

Comment:
This template seemed to be useless

Reply:
The only possible difference -- performance. Better to check Performance Tests results.